src/commcare_cloud/ansible/deploy_http_proxy.yml

```diff
@@ -11,14 +11,10 @@
         options: 'no-query no-digest'
     pri_domain_name: "{{ internal_domain_name }}"
   roles:
-    - role: squid
-      tags:
-        - 'squid'
+    - {role: squid, tags: 'squid', when: http_proxy_address|default(false)}
```

Comment:
I don't think this needs the `when` condition since it will only get applied to hosts in the `squid` group which mostly doesn't exist.

Reply:
What do you mean? Is it not important to do things to the `squid` group?

Reply:
@snopoke searching our environments for `[squid]`, I'm seeing that _no_ environments set it. Should we just remove squid-related code from here entirely?

Reply:
Merging this but gentle reminder that this question is here! @snopoke